requirements.txt

```diff
@@ -12,7 +12,7 @@ pydantic>=1.10.1
 python-socketio>=5.7.1
 requests>=2.24.0
 simplejson>=3.17.6
-sqlalchemy>=1.4.37
+sqlalchemy>=1.4.37,<=1.4.41
```

Comment:
@Aravind-psiog is it possible to make the upper limit `<2.0.0` since anything in 1.4.x should contain bugfixes

Reply:
@wjcunningham7 Changes are done.